src/main/java/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep.java

```diff
@@ -100,7 +100,7 @@ public String getFunctionName() {
         @Override
         @Nonnull
         public String getDisplayName() {
-            return "office365ConnectorSend";
+            return "Send build notifications to Microsoft Teams or Outlook";
```

Comment:
how bout "to Office365" as I believe that MS Teams and Outlook are not the only one which accepts hooks?

Reply:
also it does not have to be 'build' notification - it can be any notification :)

Reply:
Thx, I've pushed a reword - please check.